workers/cpp_physics/physics.cc

```diff
@@ -87,7 +89,7 @@ int main(int argc, char** argv) {
         // Update position of entity.
         angle += 0.5f;
         improbable::Position::Update position_update;
-        position_update.set_coords({sin(angle) * 10, 0.0, cos(angle) * 10});
+        position_update.set_coords({sin(angle) * 100, 0.0, cos(angle) * 100});
```

Comment:
Drive by:
- `std::sin` and `std::cos`.
- `100.0f` instead of `100` (but `0.0` is correct, not `0.0f`... this code is slightly odd in that it's computing the coordinates in single precision even though the `Position` component uses double position, but there you go. I would have just made `angle` a `double`, but no pressing need to change it.)

Reply:
Changed angle to double.

Reply:
Nit: still change `100` to `100.0` in that case. (And still add `std::` prefix to `sin`, `cos`.)